exaudfclient/base/javacontainer/script_options/test/converter_test.cc

```diff
@@ -33,23 +35,25 @@ INSTANTIATE_TEST_SUITE_P(
 
 
 
-class ConverterV2JarTest : public ::testing::TestWithParam<std::pair<std::string, std::set<std::string>>> {};
+class ConverterV2JarTest : public ::testing::TestWithParam<std::pair<std::string, std::vector<std::string>>> {};
 
 TEST_P(ConverterV2JarTest, jar) {
-    const std::pair<std::string, std::set<std::string>> option_value = GetParam();
+    const std::pair<std::string, std::vector<std::string>> option_value = GetParam();
     const std::string jar_option_value = option_value.first;
     std::cerr << "DEBUG: " << jar_option_value << std::endl;
 
     ConverterV2 converter;
     converter.convertExternalJar(option_value.first);
-    ASSERT_EQ(converter.getJarPaths(), option_value.second);
+    std::vector<std::string> result;
+    converter.iterateJarPaths([&](auto jar) {result.push_back(jar);});
+    ASSERT_EQ(result, option_value.second);
 }
 
-const std::vector<std::pair<std::string, std::set<std::string>>> jar_escape_sequences =
+const std::vector<std::pair<std::string, std::vector<std::string>>> jar_escape_sequences =
```

Comment:
The name of the variable doesn't fit the content

Reply:
Good catch. Fixed